workshops/beatmaker/README.md

```diff
@@ -0,0 +1,372 @@
+
+The major thing you will notice when we run our code this time is that our content has been aligned to the center.
+
+We want to style the header to give it spaces and then create three rows and four columns to correctly align the pads and give them a box or container look. We will do just that.
+
+```css
+
+h1 {
+
+color: #000;
+
+font-size: 5vw;
+
+letter-spacing: 6px;
+
+}
+
+.pad {
+
+width: 500px;
+
+display: flex;
+
+justify-content: space-between;
+
+flex-wrap: wrap;
+
+}
+
+.box {
+
+width: 100px;
+
+height: 100px;
+
+margin: 10px  0;
+
+box-shadow: 0  8px  6px  -6px  black;
+
+background-color: #444;
+
+display: flex;
+
+justify-content: center;
+
+align-items: center;
+
+font-size: 20px;
+
+color: rgba(255, 255, 255, 0.4);
+
+user-select: none;
+
+border: 4px solid;
+
+}
+```
+
+If you run your code, nothing has changed. That's because we are using classes to group our HTML elements. Let's go back to our `index.html` and to our `<div>` we will want to add the class name `box` to our nested `div` elements, and the parent header will be given the class name `pad`:
+
+```html
+
+<div  class="pad">
+
+<div  class="box">A</div>
+
+<div  class="box">B</div>
+
+<div  class="box">C</div>
+
+<div  class="box">D</div>
+
+<div  class="box">E</div>
+
+<div  class="box">F</div>
+
+<div  class="box">G</div>
+
+<div  class="box">H</div>
+  
+<div  class="box">I</div>
+
+<div  class="box">J</div>
+
+<div  class="box">K</div>
+
+<div  class="box">L</div>
+
+</div>
+
+```
+
+![Preview of HTML with layout CSS applied, the pads are in a grid but all gray](https://cloud-edj42rbl8.vercel.app/0screencapture-drum-emmanuel39hanks-repl-co-2020-11-07-23_43_18.png)
+
+And when we rerun our code, it should look like this.
+
+We will write code to make hovering effects, inactive or active states, to make it stand out. and make the experience better. We will do that with the following code
+
+```css
+.box {
+
+width: 100px;
+
+height: 100px;
+
+margin: 10px  0;
+
+box-shadow: 0  8px  6px  -6px  black;
+
+background-color: #444;
+
+display: flex;
+
+justify-content: center;
+
+align-items: center;
+
+font-size: 20px;
+
+color: rgba(255, 255, 255, 0.4);
+
+user-select: none;
+
+// We will add these two properties to the .box class to add a colored rounded solid border.
+border: 4px solid #E5446D; 
+
+border-radius:15px;
+
+}
+
+.box:hover {
+
+background-color: lighten(#444, 10%);
+
+cursor: pointer;
+
+}
+
+.box:active {
+
+background-color: darken(#444, 10%);
+
+transform: scale(1.1);
+
+transition: all  0.2s;
+
+}
+
+.active {
+
+background-color: darken(#444, 10%);
+
+transform: scale(1.1);
+
+transition: all  0.2s;
+
+}
+
+```
+
+![Preview of HTML with our finished layout CSS applied, and our pad borders colored](https://cloud-ms6ubs6qu.vercel.app/0screencapture-drum-emmanuel39hanks-repl-co-2020-11-07-23_20_37.png)
+
+We are done with our styling and layout.
```

Comment:
Yeah you REALLY need to start explaining what code does. Break it down line by line to make sure the reader understands it.